### PR TITLE
Chore: Backend - Replace Block Escaping with Escaper

### LINKED ISSUE #37099: [Issue] Chore: Backend - Replace Block Escaping with Escaper
This issue is automatically created based on existing pull request: magento/magento2#37090: Chore: Backend - Replace Block Escaping with Escaper

---------
### Description (*)
Refactors the `Magento_Backend` module to replace `$block` escaping functions with `$escaper` escaping functions.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [x] All automated tests passed successfully (all builds are green)

### PR DESCRIPTION
**Pull request by pykettk**
### Description (*)
Refactors the `Magento_Backend` module to replace `$block` escaping functions with `$escaper` escaping functions.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [x] All automated tests passed successfully (all builds are green)

### Resolved issues:
1. [x] resolves magento/magento2#37099: Chore: Backend - Replace Block Escaping with Escaper

### CODE REVIEW

**Comment by pykettk** [comment]
@magento create issue

File: app/code/Magento/Backend/view/adminhtml/templates/admin/access_denied.phtml [hunk 1]
```diff
@@ -3,29 +3,31 @@
...
-/**
- * @see \Magento\Backend\Block\Denied
- */
+declare(strict_types=1);
+
+use Magento\Backend\Block\Denied;
+use Magento\Framework\Escaper;
+
+/** @var Escaper $escaper */
+/** @var Denied $block */
```
Comment (andrewbess) [inline app/code/Magento/Backend/view/adminhtml/templates/admin/access_denied.phtml:12]: It seems that the mention of the block is redundant here.

```suggestion
use Magento\Framework\Escaper;

/** @var Escaper $escaper */
```

**Review by andrewbess — CHANGES_REQUESTED** [review]
Hello @pykettk 
Thank you for the contribution here.
Please check my comments

File: app/code/Magento/Backend/view/adminhtml/templates/admin/access_denied.phtml (hunk 1 above)
Comment (pykettk) [reply app/code/Magento/Backend/view/adminhtml/templates/admin/access_denied.phtml:12]: I'm personally a fan of keeping the `$block` reference - it helps people identify which block class the template is married to without having to dig through layout XML. I think keeping it adds value but am keen to hear your thoughts, @andrewbess 🙂

**Comment by pykettk** [comment]
Replaced by #37418

**Pull request closed without merging**